package.json

```diff
@@ -53,6 +53,7 @@
     "yargs": "^11.1.1"
   },
   "scripts": {
+    "lint": "eslint .",
```

Comment:
This looks similar to e.g. PR #8415, which was rejected. Furthermore, *if* this PR is deemed generally acceptable then this should be `gulp lint` and nothing else.

Reply:
I think that https://github.com/mozilla/pdf.js/pull/8415#issuecomment-302249805 sums it up quite nicely, and besides `gulp lint` shouldn't be more difficult to type than `npm lint` :-), hence WONTFIX seems like the appropriate resolution for this PR too.